src/librustdoc/clean/mod.rs

```diff
@@ -2069,13 +2069,13 @@ impl Clean<Item> for hir::Variant<'_> {
     }
 }
 
-impl Clean<ImplPolarity> for ty::ImplPolarity {
-    fn clean(&self, _: &DocContext<'_>) -> ImplPolarity {
+impl Clean<bool> for ty::ImplPolarity {
+    fn clean(&self, _: &DocContext<'_>) -> bool {
```

Comment:
```suggestion
    /// Returns whether the impl has negative polarity
    fn clean(&self, _: &DocContext<'_>) -> bool {
```

Reply:
I wondered for a while if I should a doc comment or not and finally didn't. But since you're suggesting it, then so be it! :D